### DISCUSSION TITLE
what is default admin email and password

### DISCUSSION THREAD

**Discussion by tandav** [open-webui/open-webui#5889] [General]
I installed open-webui using docker compose and cant log in first time. The doc only says how to reset the password.

UPD: I clicked sign up instead login and entered random email and password. It is not clear in the documentation that you have create new account.

**Comment by VINTX2** [comment]
I do not know about a default user account.

When I used open-webui the first time, I just created an account and used that to login.

**Reply by zgldh** [reply]
When I used open-webui the first time, it just bring me to http://localhost:3999/auth and I never have change to create account.

--------
UPDATED:

Finally I checked the `docker-compose.yml`  
```
    environment:
      - 'OLLAMA_BASE_URL=http://ollama:11434'
      - 'WEBUI_SECRET_KEY='
      - 'ENABLE_SIGNUP=true' <------ Change to true
```

**Reply by OmarElghamry1** [reply]
how did you find docker-compose.yml

**Reply by zgldh** [reply]
> how did you find docker-compose.yml

Just go to this page, search `docker-compose`

**Reply by snoopdujour** [reply]
What page are you talking about?

**Reply by OmarElghamry1** [reply]
yeah, what page, could you be more specific?

**Reply by zgldh** [reply]
Sorry I forgot the link https://github.com/open-webui/open-webui/blob/main/docker-compose.yaml   
I think I saw the environment variable name `ENABLE_SIGNUP` from the doc https://docs.openwebui.com/getting-started/advanced-topics/env-configuration#enable_signup 

So I tried to put it in the yaml file.

**Comment by DennisDyallo** [comment]
Agree, this whole having to create an account is confusing and should not be the default and first instruction in the README.md

**Comment by IAmMSFTSam** [comment]
This is outdated now. When running the docker container now per the QuickStart, signup requires admin authorization.

**Comment by codeliger** [comment]
The fact i have to come here instead of the docs is fucking insane

**Comment by Clearmist** [comment]
So what's the default user account credentials? I just ran `podman run -d --name openwebui -p 3000:8080 -v open-webui:/app/backend/data ghcr.io/open-webui/open-webui:main` and am being forced to log in. There is no option for creating an account.

**Reply by biondogs** [reply]
I am seeing the exact same behavior 
Edit:
Just found in the docker guide you can start with -e WEBUI_AUTH=False to skip the login
